### ISSUE TITLE
Don't work on ruby 3.0

### ISSUE THREAD

**Issue by zumkorn** [vasilakisfil/SimpleAMS#43]
Serializer
```ruby
class UserSerializer
  include SimpleAMS::DSL
  adapter SimpleAMS::Adapters::AMS
  attribute :id
end
```
Usage
```ruby
SimpleAMS::Renderer.new(user).as_json
```
Error
```ruby
ArgumentError (missing keywords: :resource, :serializer)
```
On Ruby 2.7.2 it's work

**Comment by vasilakisfil** [comment]
Hi, thanks for reporting. I ll take a look on that the following weekend and fix it.

**Comment by vasilakisfil** [comment]
Fixed by #44 . Published a new version (0.2.6). Thanks!
